modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -202,7 +206,6 @@ get.parameter.samples <- function(settings,
                                                env.samples, ens.sample.method, param.names)
     }
   }
-  
```

Comment:
nit: I suggest re-adding the empty line here -- It seems to be accepted R style  to have whitespace before the final (and thus implicitly returned) statement of a function.

Reply:
Thanks a lot！